src/stb_ds.h

```diff
@@ -1158,7 +1168,12 @@ size_t stbds_hash_bytes(void *p, size_t len, size_t seed)
     hash += (hash << 31);
     hash = (~hash) + (hash << 18);
     return hash;
-  } else {
+#else
+  else if (len == 8) {
```

Comment:
Do we need a 32-bit path for this?

Reply:
All the hash keys should be uint64_t so I think it's okay